### DISCUSSION TITLE
Distance measuring frozen?

### DISCUSSION THREAD

**Discussion by KaiBVG** [flybywiresim/aircraft#3389] [General]
Hi, in the morning I loaded the latest developer FBW build and I recognized that the "DIST"-measuring in the CDU doesn't count anymore. 
![Unbenannt-1](https://user-images.githubusercontent.com/77202222/107146906-41108f00-694b-11eb-9fe4-a2e6065f95a2.jpg)

**Comment by vblacken** [comment]
yes, the same thing happened to me, I also happened that when I loaded the route, several points were bugged and didn't show up

**Reply by Koenig70** [reply]
Same here. It just happened today in the morning after downloading the latest build.
Cheers

**Reply by Hansvanwijhe** [reply]
Same is valid for the distance-line in the MCDU. UTC and DIST are frozen. (This is in the CFBW by the way)

**Comment by sidnov** [comment]
Hey guys! 

This is a known issue currently being worked upon by the devs. Should be available once it is ready and tested for quality.
Follow this PR for more updates: #3396

**Reply by Hansvanwijhe** [reply]
Thanks for this information.
